test/e2e/annotations/affinity.go

```diff
@@ -223,4 +223,36 @@ var _ = framework.IngressNginxDescribe("Annotations - Affinity/Sticky Sessions",
 		Expect(resp.Header.Get("Set-Cookie")).Should(ContainSubstring(fmt.Sprintf("Expires=%s", expected)))
 		Expect(resp.Header.Get("Set-Cookie")).Should(ContainSubstring("Max-Age=259200"))
 	})
+
+	It("should work with use-regex annotation and session-cookie-path", func() {
```

Comment:
What about when use-regex is set but session-cookie-path is not? I'm thinking we should log an error in that case, but open to other suggestions.

Reply:
I added this warning message and test on the lua side of things. But it also could be done on the controller side. LMK what you think @ElvinEfendi